packages/keyring-controller/CHANGELOG.md

```diff
@@ -6,6 +6,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [8.0.3]
+### Uncategorized
+- Add `opts` argument to `signTransaction` ([#1789](https://github.com/MetaMask/core/pull/1789))
```

Comment:
- It looks like both of these are changes, so this needs a section name
- 1789 contains a bit more detail on this change, so we can copy from that PR

``` suggestion
### Changed
- `signTransaction` now accepts an optional `opts: Record<string, unknown>` argument to support `signTransaction` from `Keyring` type ([#1789](https://github.com/MetaMask/core/pull/1789))